packages/devtools/lib/src/framework/framework.dart

```diff
@@ -327,22 +327,23 @@ abstract class Screen {
     @required this.name,
     @required this.id,
     this.iconClass,
-    this.disabled = false,
     this.disabledTooltip = 'This screen is not available',
-  }) : helpStatus = createLinkStatusItem(
+    bool disabled,
+  })  : helpStatus = createLinkStatusItem(
           span()
             ..add(span(text: '$name', c: 'optional-700'))
             ..add(span(text: ' Docs')),
           href: 'https://flutter.github.io/devtools/$id',
           title: 'Documentation on using the $name page',
-        );
+        ),
+        disabled = allTabsEnabledByQuery ? false : disabled ?? false;
 
   final String name;
   final String id;
   final String iconClass;
   final StatusItem helpStatus;
-  final bool disabled;
   final String disabledTooltip;
+  final bool disabled;
```

Comment:
nit: you put it back one line bellow where it was originally. Not very important but I recommend using `git gui` to cleanup these sorts of no-op changes when polishing CLs.

Reply:
I did this on purpose to keep it aligned with the order in the constructor. What's flutter style's recommended param order in a constructor like this? Intuitively, I thought
```
this.name,
this.id,
this.iconClass
bool disabled = false,
this.disabledTooltip,
```
looked weird because all the "this dots" weren't together. So I tweaked the order to have all the `this.paramName`s together, and I put `bool disabled = false` at the end.

```
this.name,
this.id,
this.iconClass
this.disabledTooltip,
bool disabled = false,
```
So that is why I switched the order of these two vars here as well.